src/binary-reader.cc

```diff
@@ -806,6 +810,7 @@ Result BinaryReader::ReadFunctionBody(Offset end_offset) {
       case Opcode::F32Load:
       case Opcode::F64Load:
       case Opcode::V128Load: {
+        ERROR_UNLESS_OPCODE_ENABLED(opcode);
```

Comment:
Any reason to not do this for every opcode, i.e. before the switch? Non-disableable opcodes will always pass, and we'll cut down on the code needed to add new disabled opcodes.

Reply:
Probably just a premature optimization. I'll take a look and see if it makes a difference.